### ISSUE TITLE
Still [NSApplication remoteCommandCenter] problems.

### ISSUE THREAD

**Issue by PetrusM** [Akemi/libmpv-objc-swift#2]
Hello,

I just compiled and ran this sample, however I got a `-[NSApplication remoteCommandCenter]: unrecognized selector sent to instance` error.

I am aware of [this post](https://github.com/mpv-player/mpv-examples/issues/29#issuecomment-586518280) and [this pull request](https://github.com/mpv-player/mpv/pull/7463) that seems to be merged in the version I use, however I still see the error.

I use latest mpv from homebrew :
```bash
% brew info mpv
mpv: stable 0.32.0 (bottled), HEAD
Media player based on MPlayer and mplayer2
https://mpv.io
/usr/local/Cellar/mpv/0.32.0_5 (30 files, 10.8MB) *
```

I confirm this version is used, as `mpv_client_api_version()` returns 65643 (1.107) in the code.

**Comment by Akemi** [comment]
since it's an mpv issue, the appropriate place to report this is the mpv issue tracker and not here. the latest version is master not 0.32. the fix you linked is not in 0.32 and only in master.

if you can still reproduce this issue report it with a crash report/stack trace and the wanted info on the mpv issue tracker please.

**Comment by PetrusM** [comment]
Thank you very much, and sorry for not posting in the right place. I don't know how I thought that PR was merged.

**Comment by Akemi** [comment]
https://github.com/mpv-player/mpv/commit/b8f2811f8d180d4c2a39aabc046440c900652bb4#diff-f87a3da611540a88ccf918b7d0059c53 

it is merged just not in the 0.32 release. see this commit history https://github.com/mpv-player/mpv/commits/v0.32.0/osdep/macos/remote_command_center.swift

**Comment by PetrusM** [comment]
Yes, sorry, I corrected my post after rereading, but aparently you did not get the update...
